docs/en_US/NAS/Cream.md

```diff
@@ -0,0 +1,102 @@
+of subnetworks in the hypernetwork. To alleviate this problem, we present a simple yet effective architecture distillation method. The central idea is that subnetworks can learn collaboratively and teach each other throughout the training
+process, aiming to boost the convergence of individual models. We introduce the concept of prioritized path, which refers to the architecture candidates exhibiting superior performance during training. Distilling knowledge from the prioritized
+paths is able to boost the training of subnetworks. Since the prioritized paths are changed on the fly depending on their performance and complexity, the final obtained paths are the cream of the crop. We directly select the most promising
+one from the prioritized paths as the final architecture, without using other complex search methods, such as reinforcement learning or evolution algorithms. The experiments on ImageNet verify such path distillation method can improve the
+convergence ratio and performance of the hypernetwork, as well as boosting the training of subnetworks. The discovered architectures achieve superior performance compared to the recent MobileNetV3 and EfficientNet families under aligned
+settings. Moreover, the experiments on object detection and more challenging search space show the generality and robustness of the proposed method.
+For more details, please refer to the paper (coming soon).
+
+## Reproduction Results
+Top-1 Accuracy on ImageNet. The training with 16 Gpus is a little bit superior than 8 Gpus.
+
+| Model (M Flops) | NNI (8Gpus) | Paper (16Gpus) | 
+| ---- |:-------------:| :-----:|
+| 14M | testing | 59.6 |
+| 42M | 65.8 | 66.5 |
+| 114M | 72.1 | 72.8 |
+| 285M | 76.7 | 77.6 |
+| 470M | 78.9 | 79.2 |
+| 600M | 79.4 | 80.0 |
+
+## Requirements
+* python >= 3.6
+* torch >= 1.2
+* torchscope
+* apex
+
+## Examples
+
+[Example code](https://github.com/microsoft/nni/tree/master/examples/nas/cream)
+
+Please run the following scripts in the example folder.
+
+## Data Preparation
+
+You need to first download the [ImageNet-2012](http://www.image-net.org/) to the folder `./data/imagenet` and move the validation set to the subfolder `./data/imagenet/val`. To move the validation set, you cloud use the following script: <https://raw.githubusercontent.com/soumith/imagenetloader.torch/master/valprep.sh> 
+
+Put the imagenet data in `./data`. It should be like following:
+
+```
+./data/imagenet/train
+./data/imagenet/val
+...
+```
+
+## Quick Start
+
+### I. Search
+
+First, build environments for searching.
+
+```
+pip install -r ./requirements.txt
+```
+
+To search for an architecture, you need to configure the parameters `flops_minimum` and `flops_maximum` to specify the desired model flops, such as [0,600]MB flops. You can specify the flops interval by changing these two parameters in `./run.sh`
+
+```

```

Comment:
like other NNI examples, do we want to share the search space and example code? https://nni.readthedocs.io/en/latest/NAS/ENAS.html

Reply:
@scarlett2018  updated. pls have a check.  #2705